storm-client/pom.xml

```diff
@@ -240,6 +240,29 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>exec-maven-plugin</artifactId>
+            <executions>
+                <execution>
+                    <configuration>
+                        <executable>python2.7</executable>
```

Comment:
I'd like the storm-client build to still work on Windows. One option would be to only run this execution when the OS is UNIX-y, e.g.
```
<profile>
    <activation>
        <os><family>unix</family></os>
    </activation>
    <plugin-config-here/>
</profile>
```

Reply:
Or even better, set the executable name to python2.7 on Linux, and python on Windows.

Reply:
@srdo I've added a profile that would execute only on Unix - I didn't add one for Windows because that would change the file path separators that we're asserting on. I can change the test to make that configureable if you want

Reply:
Thanks, it looks good. I'm wondering if we can get away with also doing `pip install mock --user` as part of the Maven run, or is that a bad idea?

Reply:
@srdo It's a little messier to do that and better to leave it up to the environment to do it's own test setup. --user is for then there's no virtualenv (which will vary from machine to machine) - which is a good example of how the host itself should do the setup.